src/SourceBuild/content/repo-projects/sdk.proj

```diff
@@ -11,9 +11,8 @@
     <!-- Propagate RID set in source-build to sdk repo -->
     <_platformIndex>$(NETCoreSdkPortableRuntimeIdentifier.LastIndexOf('-'))</_platformIndex>
     <_baseOS>$(NETCoreSdkPortableRuntimeIdentifier.Substring(0, $(_platformIndex)))</_baseOS>
-    <_baseOS Condition="'$(OverrideTargetOS)' != ''">$(OverrideTargetOS)</_baseOS>
+    <_baseOS Condition="'$(PortableBuild)' == 'true'">$(TargetOS)</_baseOS>
     <_targetPortableArch>$(TargetArchitecture)</_targetPortableArch>
-    <_targetPortableArch Condition="'$(OverrideTargetArch)' != ''">$(OverrideTargetArch)</_targetPortableArch>
 
     <BuildArgs>$(BuildArgs) /p:PackageProjectUrl=https://github.com/dotnet/sdk</BuildArgs>
     <BuildArgs>$(BuildArgs) /p:PortableRid=$(_baseOS)-$(_targetPortableArch)</BuildArgs>
```

Comment:
Shouldn't this just be `PortableRid` from https://github.com/dotnet/dotnet/blob/dd89635deed60d1c113e31ab5105b1349e8ad1b9/Directory.Build.props#L53-L58?